pyseir/inference/model_fitter.py

```diff
@@ -308,52 +339,105 @@ def _fit_seir(self, R0, t0, eps, t_break, test_fraction, hosp_fraction, log10_I_
 
         return chi2_deaths + chi2_cases + chi2_hosp
 
+    def get_posterior_estimate_eps(self, R0, eps, eps_error, plot=False):
+        """
+        Generate a posterior estimate for epsilon based on the inferred R0. This
+        is a little weird right now since we actually want a prior on Reff. So
+        in this case we use the inferred R0 to convert eps -> Reff, apply a
+        prior, and invert this transform to get back to the epsilon Max
+        A-Posteriori (MAP) estimate.
+
+        Returns
+        -------
+        posterior_map_estimate: float
+            Max A-Posteriori (MAP) estimate for epsilon.
+        """
+        R_eff = R0 * eps
+        R_eff_stdev = R0 * eps_error
```

Comment:
This seems to miss the error of R0.

Reply:
That is correct.  We need a better formalism to get the joint posteriors.

Reply:
Ok, should be fine this is temporary solution, and agreed better formalism is needed.